### PR TITLE
Added documentation for installing the front-end side

### PR DESCRIPTION
**Pull request by ahmadajmi**
Part of the #29 issue for installing the front-end side

### CODE REVIEW

File: npm-debug.log [hunk 1]
```diff
@@ -0,0 +1,20 @@
+0 info it worked if it ends with ok
```
Comment (montaro) [inline npm-debug.log]: Log file should be added to the repo?

Comment (ahmadajmi) [inline npm-debug.log]: I think it shouldn't

Comment (montaro) [inline npm-debug.log]: I would suggest you remove it from the repo and push again to update the pull request.

Comment (ahmadajmi) [inline npm-debug.log]: OK done

Comment (ikhattab) [inline npm-debug.log]: It would be better if this file is added to .gitignor too

Comment (ahmadajmi) [inline npm-debug.log]: @ikhattab Added in the last commit as @montaro suggested.

**Pull request closed without merging**